src/generator.js

```diff
@@ -33,7 +33,7 @@ function generator(fn, argArray, options) {
             }
 
             beforeFormat.call(this);
-            return React.DOM.span(null, afterFormat.call(this, this.format()));
+            return React.DOM.span(componentProps, afterFormat.call(this, this.format()));
```

Comment:
Yeap, that's the first thing that occurred me reviewing it. To filter the react-globalize props out. But, I agree it shouldn't harm leaving it as is.